ext/AvizExt/AvizExt.jl

```diff
@@ -302,16 +306,7 @@ function ADRIA.viz.explore(rs::ResultSet)
     heatmap!(ft_import, sensitivities)
     Colorbar(layout.importance[1, 2]; colorrange=(0.0, 1.0))
 
-    # TODO: Separate this out into own function
-    # Make temporary copy of GeoPackage as GeoJSON
-    tmpdir = mktempdir()
-
-    local geo_fn = joinpath(tmpdir, "Aviz_$(rs.name).geojson")
-    try
-        GDF.write(geo_fn, rs.site_data; driver="geojson")
-    catch
-        GDF.write(geo_fn, rs.site_data; geom_columns=(:geom,), driver="geojson")
-    end
+    geo_fn = make_geojson_copy(rs)
```

Comment:
So previously the user had to provide their own geojason but now this is created directly from the geopackage?

Reply:
Yes.

In the future the whole conversion to GeoJSON will be unnecessary, this is a temporary workaround for now.
There's a change in #382 which simplifies things even further.

Reply:
Cool, thanks